snap/README.md

```diff
@@ -96,53 +107,46 @@ Enable rules engine, to create streams and sinks:
 
-API Gateway is needed to enter UI secure mode. Before the API Gateway can be used, a user and group must be created, and a JWT access token generated. This can be accomplished via the `secrets-config` command, or by using `snap set` commands
+API Gateway is needed to enter UI secure mode. Before the API Gateway can be used, a user and group must be created, and a JWT access token generated. This can be accomplished via the `secrets-config` and  the `snap set` commands:
 
-The first step is to add a user. You need to create a public/private keypair, which can be done with:
+1. Generate a public/private keypair
 
 ```
 $ openssl ecparam -genkey -name prime256v1 -noout -out private.pem
 $ openssl ec -in private.pem -pubout -out public.pem
 ```
 
-The second step is if you then create the user using the secrets-config command, then you need to provide:
-
-- The username
-- The public key
-- (optionally) ID. This is a unique string identifying the credential. It will be required in the next step to create the JWT token. If you don't specify it, then an auto-generated one will be output by the secrets-config command
-
-Do this using `snap set`:
+2. Configure user and public-key
 
 ```bash
 $ sudo snap set edgexfoundry env.security-proxy.user=user01,USER_ID,ES256
 $ sudo snap set edgexfoundry env.security-proxy.public-key="$(cat public.pem)"
 ```
 
-The third step is then to generate a token using the user ID which you specified:
+3. Generate a token
 
 ```bash
 $ edgexfoundry.secrets-config proxy jwt --algorithm ES256 --private_key private.pem --id USER_ID --expiration=1h
 ```
 
-An output has a long alphanumeric sequence of the format < alphanumeric >.< alphanumeric >.< alphanumeric >:
+An output has a long alphanumeric sequence of the format `<alphanumeric>.<alphanumeric>.<alphanumeric>`:
 
 ```
 eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJVU0VSX0lEIiwiZXhwIjoxNjM1OTM1NjYxLCJuYmYiOjE2MzU5MzIwNjEsImlhdCI6MTYzNTkzMjA2MX0.Kzsu_3L7wVNbjAbGfNatmyVtsmDaodOG_b5nCHKywU327xdfqpgno1g3ai1yu8hz6pG0_2eHnordCSWxaPFoMA
 ```
 
-This output is the JWT token for login UI secure mode. This token will be asked again if you exit the web page, and re-enter it. Please write the token you generated down in case you re-enter the UI web page. The needed token can not be generated for the second time
+This output is the JWT token for UI login in secure mode. Please keep the token in a safe place for future re-use as the same token cannot be regenerated or recovered from EdgeX's secret-config CLI. The token is required each time you reopen the web page.
 
 ### Using the edgex-ui snap
 
-Open your browser [http://localhost:4000](http://localhost:4000/):![img](https://lh5.googleusercontent.com/OzQFePPICpiaMTxEm8cnxaCFn7ageCbLR9J5TpzkGtmRIip-pB02D7H994gKvVS2CTD3YChngX72SGtaLxEVjdC4j9bgYTFCGHvRfzr25i-qIk3UNMprucpPa2oBpgI_W7KZVJ4n)
+Open your browser [http://localhost:4000](http://localhost:4000/):![](/home/mengyi/Desktop/EdgeX_14/edgex-ui-go/snap/local/assets/ui-login.jpg)
```

Comment:
Wrong image path.

Reply:
I have updated this path. This commit you reviewed was out of date.